### PR TITLE
fix(seed): Add tests, solution, and image for Project Euler problem 15

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Added more tests, a solution, and an image to make the description more clear. 

The image is from Project Euler but is re-hosted on imgur. Should be fine to use the image under the Creative Commons Licence, but if not, I could recreate it and update this PR.

BREAKING CHANGE: None

Here's the solution for QA:

```
function latticePaths(gridSize) {
  let paths = 1;
  
  for (let i = 0; i < gridSize; i++) {
    paths *= (2 * gridSize) - i;
    paths /= i + 1;
  }
  return paths;
}
```

### CODE REVIEW

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json [hunk 1]
```diff
@@ -572,21 +572,26 @@
...
       "challengeSeed": [
-        "function euler15() {",
+        "function latticePaths(gridSize) {",
         "  // Good luck!",
         "  return true;",
         "}",
         "",
-        "euler15();"
+        "latticePaths(4);",
+        "latticePaths(9);",
```
Comment (Bouncey) [inline seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: @scissorsneedfoodtoo ideally, we only want to call the challenge function in the seed a maximum of one time

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json]: @Bouncey, I'm sorry about that. I can go back and change that in this and my other PRs.

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks @scissorsneedfoodtoo 👍

**Comment by scissorsneedfoodtoo** [comment]
@Bouncey, thank you for taking the time to QA and merge this!

**Pull request merged**